utils/jwtUtil.js

```diff
@@ -80,12 +80,12 @@ function verifyToken(req, cb) {
 }
 
 /**
- * Verify jwt token.
+ * Get token details: username, token name from Token.
  * @param  {object}   req - request object
  * @param  {Function} cb - callback function
  * @returns {User}
  */
-function getUsernameFromToken(req) {
+function getTokenDetailsFromToken(req) {
   return new Promise((resolve, reject) => {
     if (req && req.headers && req.headers.authorization) {
```

Comment:
Nice, and this function gets called in 'logAndRemoveJobOnComplete function ok' as well as an integration test.